cockroach-admin/types/src/node.rs

```diff
@@ -0,0 +1,100 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! Node-related types for the CockroachDB Admin API.
+
+pub use cockroach_admin_types_versions::latest::node::*;
+
+/// CockroachDB Node ID (internal representation)
+///
+/// This field is stored internally as a String to avoid questions
+/// about size, signedness, etc - it can be treated as an arbitrary
+/// unique identifier.
+///
+/// Note: This is an internal type used for CLI parsing and internal
+/// storage, distinct from the published API `NodeId` type in the
+/// versions crate which has a different structure.
```

Comment:
Sorry if this is obvious and I'm missing it - why do we have separate `NodeId` and `InternalNodeId` now?